Samples/resource-owner-password-credential/Angular2/openiddict-angular2-server/src/openiddict-angular2/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Builder;
+using openiddict_angular2.Models;
+
+namespace openiddict_angular2.Controllers
```

Comment:
Consider renaming the namespace following the C# Coding guidelines [here](https://msdn.microsoft.com/en-us/library/ms229026(v=vs.110).aspx)